cernopendata/modules/theme/static/oai2.xsl

```diff
@@ -0,0 +1,658 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
this is something each service needs to implement?

Reply:
I'm following the example of zenodo. Indeed, it would be better if it was provided by invenio-oaiservice. I'm happy to move it there if you prefer

Reply:
I am not familiar with the usage of this but if it something every instance will copy, then moving it there make sense.